crates/compiler/collections/src/all.rs

```diff
@@ -30,9 +30,6 @@ pub type SendMap<K, V> = im::hashmap::HashMap<K, V, BuildHasher>;
 
 pub type SendSet<K> = im::hashset::HashSet<K, BuildHasher>;
 
-// pub type BumpMap<'a, K, V> = hashbrown::HashMap<K, V, BuildHasher, hashbrown::BumpWrapper<'a>>;
```

Comment:
The newest hashbrown no longer supports `BumpWrapper`, so uncommenting this code would not work anymore.